sql/migrations/20210515110157_world.sql

```diff
@@ -0,0 +1,837 @@
+(5477, 25, -10239.630859375, -2988.478515625, 22.00403594970703125, 100, 0, 0),
+(5477, 26, -10219.9150390625, -2974.809814453125, 19.83499908447265625, 100, 0, 0),
+(5477, 27, -10189.126953125, -2962.115478515625, 22.09839057922363281, 100, 0, 0),
+(5477, 28, -10178.8720703125, -2963.053955078125, 23.43799018859863281, 100, 0, 0),
+(5477, 29, -10189.126953125, -2962.115478515625, 22.09839057922363281, 100, 0, 0),
+(5477, 30, -10219.9150390625, -2974.809814453125, 19.83499908447265625, 100, 0, 0),
+(5477, 31, -10239.5234375, -2988.482666015625, 22.01819610595703125, 100, 0, 0),
+(5477, 32, -10272.6298828125, -2989.186767578125, 22.01354026794433593, 100, 0, 0),
+(5477, 33, -10294.0673828125, -2991.93359375, 22.43688011169433593, 100, 0, 0),
+(5477, 34, -10300.271484375, -3007.030029296875, 21.12286567687988281, 100, 0, 0),
+(5477, 35, -10319.2578125, -3044.580810546875, 22.30823516845703125, 100, 0, 0),
+(5477, 36, -10331.9814453125, -3065.276123046875, 23.26966094970703125, 100, 0, 0),
+(5477, 37, -10309.5048828125, -3074.629150390625, 23.08895301818847656, 100, 0, 0),
+(5477, 38, -10282.701171875, -3092.743896484375, 23.1742401123046875, 100, 0, 0),
+(5477, 39, -10240.646484375, -3097.669189453125, 21.9894256591796875, 100, 0, 0),
+(5477, 40, -10202.677734375, -3103.907470703125, 24.04048919677734375, 100, 0, 0),
+(5477, 41, -10168.91015625, -3113.720703125, 22.32606887817382812, 100, 0, 0),
+(5477, 42, -10152.5146484375, -3146.046142578125, 20.62872314453125, 100, 0, 0),
+(5477, 43, -10116.873046875, -3159.724853515625, 23.36187744140625, 100, 0, 0),
+(5477, 44, -10111.1806640625, -3191.95947265625, 20.99517822265625, 100, 0, 0),
+(5477, 45, -10135.18359375, -3221.2890625, 22.11031723022460937, 100, 0, 0),
+(5477, 46, -10151.513671875, -3240.291259765625, 20.44405174255371093, 100, 0, 0),
+(5477, 47, -10179.041015625, -3269.541015625, 22.29747200012207031, 100, 0, 0),
+(5477, 48, -10212.1845703125, -3288.988525390625, 20.03130912780761718, 100, 0, 0),
+(5477, 49, -10208.2080078125, -3308.25341796875, 22.02155876159667968, 100, 0, 0),
+(5477, 50, -10210.7939453125, -3339.044677734375, 24.30500602722167968, 100, 0, 0),
+(5477, 51, -10212.0126953125, -3382.2548828125, 23.25324440002441406, 100, 0, 0),
+(5477, 52, -10217.236328125, -3422.067138671875, 20.74958229064941406, 100, 0, 0),
+(5477, 53, -10218.26171875, -3447.06640625, 22.42780876159667968, 100, 0, 0),
+(5477, 54, -10208.9814453125, -3470.27783203125, 23.36289024353027343, 100, 0, 0),
+(5477, 55, -10199.28125, -3489.0244140625, 22.89615440368652343, 100, 0, 0),
+(5477, 56, -10174.763671875, -3505.798583984375, 23.74078178405761718, 100, 0, 0);
+INSERT INTO `creature_groups` (`leader_guid`, `member_guid`, `dist`, `angle`, `flags`) VALUES
+(43588, 43589, 4, 2.35619, 11),
+(43588, 43590, 4, 3.92699, 11);
+
+-- Fix Gnomish Battle Chicken (credit cmangos)
+UPDATE `spell_template` SET `effect1`= 42 WHERE `entry` = 31366;

```

Comment:
What is going on here? Spell 31366 called Root Anybody Forever.
https://classicdb.ch/?spell=31366

And you are changing it's effect from Apply Aura (6) to Summon Guardian (42) ?  And why are you editing spell_template in the first place, as this data comes from dbc ?